assets/js/components/adminbar/common.stories.js

```diff
@@ -26,7 +26,14 @@ import { MODULES_ANALYTICS } from '../../modules/analytics/datastore/constants';
 import { getAnalyticsMockResponse } from '../../modules/analytics/util/data-mock';
 import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
 import { provideSearchConsoleMockReport } from '../../modules/search-console/util/data-mock';
-import { replaceValuesInAnalyticsReportWithZeroData } from '../../../../.storybook/utils/zeroReports';
```

Comment:
All of the changes to this file can be reverted as you've moved them to `common-GA4.stories.js`.

Reply:
Good catch. Thanks 👍